ratios/ratio-capacite-remboursement.md

```diff
@@ -39,23 +38,23 @@ Comme leur nom l’indique, les produits et les charges non encaissables sont de
 
 On retrouve notamment dans les charges non encaissables :
 
-* les dotations aux amortissements ;
-* les dotations aux dépréciations et aux provisions.
+- les dotations aux amortissements ;
+- les dotations aux dépréciations et aux provisions.
 
 On retrouve notamment dans les produits non encaissables :
 
-* les subventions perçues sur investissements ;
-* les reprises sur amortissements et provisions.
+- les subventions perçues sur investissements ;
+- les reprises sur amortissements et provisions.
 
 ## Intérêt du ratio pour la banque
 
 Les banquiers utilisent la CAF pour mettre en regard les excédents de trésorerie d’une entreprise avec ses dettes, grâce au calcul du ratio de capacité de remboursement (dettes/CAF).
 
-Le ratio de capacité de remboursement donne aux banquiers le nombre d’années que mettrait votre entreprise à rembourser ses dettes — si ces dettes n’augmentaient pas, et que votre CAF restait la même au cours des prochaines années. Il renseigne votre banquier sur la capacité de votre entreprise à dégager de la trésorerie, et donc,* in fine*, à rembourser ses crédits — car votre entreprise pioche dans sa trésorerie pour rembourser ses prêts.
+Le ratio de capacité de remboursement donne aux banquiers le nombre d’années que mettrait votre entreprise à rembourser ses dettes — si ces dettes n’augmentaient pas, et que votre CAF restait la même au cours des prochaines années. Il renseigne votre banquier sur la capacité de votre entreprise à dégager de la trésorerie, et donc,_ in fine_, à rembourser ses crédits — car votre entreprise pioche dans sa trésorerie pour rembourser ses prêts.
```

Comment:
Italic doesn’t seem to work here, the space before "in" should be removed.

Reply:
Thanks. I'll fix that in a new commit. 👍